components/role-mgt/org.wso2.carbon.identity.role.v2.mgt.core/src/main/java/org/wso2/carbon/identity/role/v2/mgt/core/dao/RoleDAOImpl.java

```diff
@@ -1379,6 +1380,40 @@ public Map<String, String> getMainRoleToSharedRoleMappingsBySubOrg(List<String>
         return rolesMap;
     }
 
+    @Override
+    public Map<String, String> getSharedRoleToMainRoleMappingsBySubOrg(List<String> roleIds,
+                                                                       String subOrgTenantDomain)
```

Comment:
Check formatting. Should be able to be in one line

Reply:
Addressed and resolved in commit: [b2054ba](https://github.com/wso2/carbon-identity-framework/pull/6162/commits/b2054baf27f17ededa73e9b12c15530489faa707)